pkg/raft/raft.go

```diff
@@ -1722,8 +1724,10 @@ func (r *raft) handleAppendEntries(m pb.Message) {
 		// committed entries at m.Term (by raft invariants), so it is safe to bump
 		// the commit index even if the MsgApp is stale.
 		lastIndex := a.lastIndex()
-		r.raftLog.commitTo(logMark{term: m.Term, index: min(m.Commit, lastIndex)})
-		r.send(pb.Message{To: m.From, Type: pb.MsgAppResp, Index: lastIndex})
+		commitIndex := min(m.Commit, lastIndex)
+		r.raftLog.commitTo(logMark{term: m.Term, index: commitIndex})
```

Comment:
nit: the `commitTo` line did not need to change (`commitIndex` doesn't have a second usage)